packages/backend/test/yeomanui.spec.ts

```diff
@@ -314,6 +315,7 @@ describe("yeomanui unit test", () => {
 
   it("_notifyGeneratorsChange", async () => {
     envUtilsMock.expects("getGeneratorsData").resolves();
+    wsConfigMock.expects("get").withExactArgs("ApplicationWizard.HideGenerator").resolves();
```

Comment:
the test passes by chance - config.get() is a sync function that should return a string value (according to it signature. 
```suggestion
    wsConfigMock.expects("get").withExactArgs("ApplicationWizard.HideGenerator").returns("");
```